index.html

```diff
@@ -718,7 +715,16 @@ <h3>Proof Configuration (eddsa-2022)</h3>
 <var>options</var>.<var>proofPurpose</var>.
             </li>
             <li>
-Return <var>proofConfig</var>.
+Set <var>proofConfig</var>.<var>@context</var> to
+<var>unsecuredDocument</var>.<var>@context</var>
```

Comment:
This might be the wrong thing to do, because it'll pull in all sorts of contexts that don't have anything to do with the signature. It's also unnnecessary if you're using something like the VC v2.0 context and is unnecessary when using something like a JCS-base canonicalization (that encapsulates JSON-LD). IOW, this step interplays in ways that are shaky for non-JSON-LD solutions. 

There are a couple of code paths we have to think about here:

1. You are NOT doing a JSON-LD-to-RDF canonicalization, which means you don't need to do anything with `@context`.
2. You are doing an JSON-LD-to-RDF canonicalization, but the existing `@context` has the necessary data integrity term mappings (such as the VC v2.0 context).
3. You are doing an JSON-LD-to-RDF canonicalization, and the existing `@context` DOES NOT have the necessary data integrity term mappings (e.g. you're signing a vanilla JSON-LD document). In this case, you want to be surgical in what you pull in (just the data integrity v1 context). In this case, should we add the data integrity v1 `@context` to the proof or at the top level of the document?

/cc @dlongley @davidlehn @gkellogg

Reply:
This is only an *attempt* to resolve the issue of the missing steps to get a hash of the proof configuration in the `cryptosuite: eddsa-2022` case. For JCS this is not needed. For the current two cases that use RDF canonicalization something is missing in the draft. I think I saw this in a couple implementations and it validates  against CHAPI playground.

Requiring a specific context be used can work to.